src/Symfony/Component/Scheduler/Messenger/ScheduledStamp.php

```diff
@@ -12,10 +12,18 @@
 namespace Symfony\Component\Scheduler\Messenger;
 
 use Symfony\Component\Messenger\Stamp\NonSendableStampInterface;
+use Symfony\Component\Scheduler\Trigger\TriggerInterface;
 
 /**
  * @experimental
  */
 final class ScheduledStamp implements NonSendableStampInterface
 {
+    public function __construct(
+        public readonly ?string $name,
+        public readonly TriggerInterface $trigger,
+        public readonly \DateTimeImmutable $runAt,
```

Comment:
Maybe we need a better name? This might not be the true run date (if the scheduler is catching up). Maybe `$expectedRunAt`?

/cc @tucksaun

Reply:
`expectedRunAt` sounds indeed better than `runAt`.

Other suggestions:
- `scheduledAt`
- `triggeredAt` (to keep the idea of the trigger)